src/overlays/ovl_i5/fox_ti.c

```diff
@@ -1,138 +1,5375 @@
+        }
+    }
+
+    if ((boss->state >= 3) && (boss->state <= 13)) {
+        gBossFrameCount++;
+    }
+    switch (boss->state) {
+        case 0:
+        case 1:
+        case 2:
+        case 3:
+        case 4:
+            func_i5_80192118(boss);
+            break;
+        case 5:
+        case 6:
+        case 7:
+        case 8:
+        case 9:
+        case 10:
+        case 11:
+        case 12:
+        case 13:
+            func_i5_80193DF0(boss);
+            break;
+    }
+
+    switch (boss->swork[30]) {
+        case 0:
+            switch (boss->swork[31]) {
+                case 1:
+                    Radio_PlayMessage(gMsg_ID_20266, RCID_FOX);
+                    break;
+                case 80:
+                    gTeamShields[2] = 1;
+                    Radio_PlayMessage(gMsg_ID_4112, RCID_SLIPPY);
+                    gTeamShields[2] = -2;
+                    break;
+                case 240:
+                    gTeamShields[2] = 1;
+                    Radio_PlayMessage(gMsg_ID_4093, RCID_SLIPPY);
+                    gTeamShields[2] = -2;
+                    break;
+            }
+
+            break;
+        case 1:
+            if (boss->swork[31] == 1) {
+                func_800182F4(0x105A00FF);
+                func_800182F4(0x115A00FF);
+            }
+            break;
+        case 2:
+            if (boss->swork[31] == 11) {
+                gTeamShields[2] = 1;
+                Radio_PlayMessage(gMsg_ID_4111, RCID_SLIPPY);
+                gTeamShields[2] = -2;
+            }
+            break;
+        case 3:
+            switch (boss->swork[31]) {
+                case 1:
+                    break;
+                case 50:
+                    if (gTeamShields[1] > 0) {
+                        Radio_PlayMessage(gMsg_ID_4094, RCID_FALCO);
+                    }
+                default:
+                    break;
+            }
+            break;
+        case 4:
+            if (boss->swork[35] == 0) {
+                if (D_i5_801BBEF0[42] == 0) {
+                    switch (boss->swork[4]) {
+                        case 1:
+                            if (gTeamShields[3] > 0) {
+                                Radio_PlayMessage(gMsg_ID_4095, RCID_PEPPY);
+                            }
+                            D_i5_801BBEF0[42] = 1;
+                            break;
+                        case 3:
+                            if ((D_i5_801BBEF0[8] == 0) && (gTeamShields[1] > 0)) {
+                                Radio_PlayMessage(gMsg_ID_4097, RCID_FALCO);
+                            }
+                            D_i5_801BBEF0[42] = 1;
+                            break;
+                        default:
+                            break;
+                    }
+
+                    if (boss->swork[4] >= 4) {
+                        if (!(D_i5_801BBEF0[46] & 1)) {
+                            if ((gTeamShields[1] > 0) && (D_i5_801BBEF0[8] == 0) && (gRadioState == 0) &&
```

Comment:
A lot of these nested if statements could probably use `||` to reduce indentation level.